app/components/GW2Results/Marketshare/index.tsx

```
@@ -162,13 +214,25 @@ const getChartData = (
         value = item.historic_price_average
         break
       case 'pricePercentChange':
-        value = Math.abs(item.pricePercentChange)
+        // For negative values, use Math.abs() to make them positive for treemap
+        const priceChange = item.pricePercentChange
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)